adapters/repos/db/shard_write_delete.go

```diff
@@ -142,26 +141,28 @@ func (s *Shard) deleteOne(ctx context.Context, bucket *lsmkv.Bucket, obj, idByte
 func (s *Shard) cleanupInvertedIndexOnDelete(previous []byte, docID uint64) error {
 	previousObject, err := storobj.FromBinary(previous)
 	if err != nil {
-		return errors.Wrap(err, "unmarshal previous object")
+		return fmt.Errorf("unmarshal previous object: %w", err)
 	}
 
 	// TODO text_rbm_inverted_index null props cleanup?
 	previousInvertProps, _, err := s.analyzeObject(previousObject)
 	if err != nil {
-		return errors.Wrap(err, "analyze previous object")
+		return fmt.Errorf("analyze previous object: %w", err)
 	}
 
-	s.subtractPropLengths(previousInvertProps)
+	if err = s.subtractPropLengths(previousInvertProps); err != nil {
+		return fmt.Errorf("subtract prop lengths: %w", err)
+	}
```

Comment:
Here is the actual fix. 

The rest of these changes are either syntactical or moving away from the now unmaintained `pkg/errors` lib